publication-commons/src/main/java/no/unit/nva/publication/model/business/ThirdPartySystem.java

```diff
@@ -0,0 +1,25 @@
+package no.unit.nva.publication.model.business;
+
+import static java.util.Arrays.stream;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum ThirdPartySystem {
+    WISE_FLOW("WISEflow"), INSPERA("Inspera"), OTHER("Other");
```

Comment:
Not sure this is the best name for this enum. Come with suggestions if you have ideas.